### PR TITLE
Workflow update

### PR DESCRIPTION
**Pull request by gokris165**
Using a Personal Access Token to trigger workflow checks after github-actions[bot] commits code.

### CODE REVIEW

File: .github/workflows/actions.yml [hunk 1]
```diff
@@ -9,167 +9,177 @@ jobs:
   # main branch and pull request branch
   changes:
     runs-on: ubuntu-latest
-    outputs:
-      core: ${{ steps.filter.outputs.core }}
-      client: ${{ steps.filter.outputs.client }}
-      transport: ${{ steps.filter.outputs.transport }}
-      server: ${{ steps.filter.outputs.server }}
-      commit-author: ${{ steps.check.outputs.author }}
+    # outputs:
```
Comment (breed) [inline .github/workflows/actions.yml:12]: you need to remove the stuff that is currently commented out.

**Review by breed — CHANGES_REQUESTED** [review]
were you able to get it working without using tokens?

File: .github/workflows/actions.yml (hunk 1 above)
Comment (gokris165) [reply .github/workflows/actions.yml:12]: Yeah that was just for cat-ing .git/config, I'll revert it back. I'm still working on getting the tokens to work, and if that doesn't work then I'll try the GitHub app approach. Until then I'll mark this PR as a draft.

**Comment by gokris165** [comment]
IT WORKS @breed

**Review by breed — APPROVED** [review]

**Pull request merged**